libs/contact/guides/drivers/in-memory-driver.md

```diff
@@ -0,0 +1,22 @@
+# Contact InMemory Driver
+
+The In-Memory driver is for rapid development without the need to set up a magento/shopify/etc backend. It will mock out the submission of a contact forum and operate like a functional backend.
+
+To set up, inside of your `app.module` make sure to include the `DaffContactInMemoryDriverModule.forRoot()` inside of your imports section, as well as to import the `DaffContactInMemoryDriverModule` from the `@daffodil/contact/testing` library and the `HttpClientInMemoryWebApiModule` from `angular-in-memory-web-api`.
```

Comment:
"inside of your `app.module` make sure to include the `DaffContactInMemoryDriverModule.forRoot()` inside of your imports section"

I think this is harder to read than it should be, because the "inside of your `app.module`" and "inside of your imports section" go together, but they're split up by "make sure to include the `DaffContactInMemoryDriverModule.forRoot()`". A possible alternative could be "include the DaffContactInMemoryDriverModule.forRoot() in the imports section of your app.module".